src/App.vue

```diff
@@ -325,7 +317,7 @@ export default class App extends Mixins(BcolMixin, DateMixin, FilingTemplateMixi
       // reset errors in case this method is invoked more than once (ie, retry)
       this.resetFlags()
 
-      // get business ID query param
+      // get and store business ID query param
```

Comment:
i admire your explicitness with regards to proper comments :) Your comments are continuously evolving!

Reply:
Haha, thanks!